beetmoverscript/task.py

```diff
@@ -109,29 +110,45 @@ def add_balrog_manifest_to_artifacts(context):
     utils.write_json(abs_file_path, context.balrog_manifest)
 
 
-def get_upstream_artifact(context, taskid, path):
```

Comment:
Hah, nice cleanup, didn't know of `scriptworker_artifacts.get_and_check_single_upstream_artifact_full_path` 👍

Reply:
I just realize I implemented it a while ago in https://github.com/mozilla-releng/scriptworker/pull/95. I knew beetmover had its own logic, but I didn't cc you on it. Sorry about this

Reply:
No, absolutely no worries, this is super nice! 👍